ui/src/personal/LastfmScrobbleToggle.js

```diff
@@ -23,7 +23,9 @@ const Progress = (props) => {
     )
     const callbackUrl = `${window.location.origin}${callbackEndpoint}`
     openedTab.current = openInNewTab(
-      `https://www.last.fm/api/auth/?api_key=${localStorage.getItem('lastfm-apikey')}&cb=${callbackUrl}`,
+      `https://www.last.fm/api/auth/?api_key=${localStorage.getItem(
+        'lastfm-apikey',
+      )}&cb=${callbackUrl}`,
```

Comment:
Was required to run prettier before I could push due to git hooks.

Reply:
Weird, but that's fine.